### PR TITLE
Verify that base profiles exist in GitHub container registry

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:
We now feature two package repositories for the baseprofiles in https://github.com/orgs/security-profiles/packages

Means that we can use `spoc push` to manually maintain them for now. At some later stage we can add automation to push them on the fly.

The repositories have to made public as well, otherwise `cosign verify` will fail.

#### Which issue(s) this PR fixes:

Part of #1482 
#### Does this PR have test?

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Review by ccojocar — APPROVED** [review]

**Comment by saschagrunert** [comment]
Blocked by https://github.com/kubernetes/org/issues/4105

**Comment by saschagrunert** [comment]
Updated to match the new org and invited y'all. PTAL @kubernetes-sigs/security-profiles-operator-maintainers

**Review by ccojocar — APPROVED** [review]

**Pull request merged**
